lampo-cli/src/main.rs

```diff
@@ -1,24 +1,23 @@
 mod args;
 
-use std::process::exit;
-
-use radicle_term as term;
-
 use lampo_client::errors::Error;
 use lampo_client::UnixClient;
 use lampo_common::error;
 use lampo_common::json;
+use radicle_term as term;
+use std::process::exit;
```

Comment:
We have an order of the imports if you will pay attention to is you will not the following orders

```
<std imports>

<external dependencies>

<lampo imports>
```